servant-client-core/src/Servant/Client/Core/Request.hs

```diff
@@ -52,9 +52,10 @@ import           Network.HTTP.Types
                  (Header, HeaderName, HttpVersion (..), Method, QueryItem,
                  http11, methodGet)
 import           Servant.API
-                 (ToHttpApiData, toEncodedUrlPiece, toHeader, SourceIO)
+                 (SourceIO, ToHttpApiData, toEncodedUrlPiece, toHeader)
```

Comment:
Hm, is changing this file really neccessary? Generally, I'd only do formatting in files where I introduce significant changes.

Reply:
`stylish-haskell` (that was enabled in my configuration) put constructors before the function.
I could disable it and undo changes in imports.

Reply:
Fixed.